library/src/main/scala/scala/scalajs/LinkingInfo.scala

```diff
@@ -108,4 +108,17 @@ object LinkingInfo {
   def assumingES6: Boolean =
     linkingInfo.assumingES6
 
+  /** Returns the version of the linker.
+   *
+   *  This ends up being constant-folded to a constant at link-time. So
+   *  constant-folding, inlining, and other local optimizations can be
+   *  leveraged with this "constant" to write polyfills that can be
+   *  dead-code-eliminated.
+   *
+   *  Used to define the `java.vm.version` property in `java.lang.System`.
+   *
+   */
+  @inline
+  def version: String =
+    linkingInfo.version
```

Comment:
Since it is accessible with `java.vm.version`, I wouldn't expose this here at all.